pkg/backends/locust/resources.go

```diff
@@ -227,9 +227,9 @@ func newWorkerJob(
 
 	ownerRef := metaV1.NewControllerRef(&loadTest, loadTestV1.SchemeGroupVersion.WithKind("LoadTest"))
 
-	imageRef := fmt.Sprintf("%s:%s", loadTest.Spec.MasterConfig.Image, loadTest.Spec.MasterConfig.Tag)
+	imageRef := fmt.Sprintf("%s:%s", image.Image, image.Tag)
 	if imageRef == ":" {
-		imageRef = fmt.Sprintf("%s:%s", image.Image, image.Tag)
+		imageRef = fmt.Sprintf("%s:%s", loadTest.Spec.MasterConfig.Image, loadTest.Spec.MasterConfig.Tag)
```

Comment:
should it be the Worker image instead ?

Reply:
Hey @lucasmdrs,

hmm, I only flipped the conditions to look at the supplied `image` argument first and then fallback to Master image to keep the behavior backwards compatible. Following the logic "if the worker is not specified use the master".

Now the question is basically what behavior is desired - the one I mentioned above or "if worker is not specified use the default"?

(sorry for late response, I was on vacation)